en/micro-integrator/docs/references/connectors/kafka-connector/kafka-inbound-endpoint-example.md

```diff
@@ -0,0 +1,88 @@
+# Kafka Inbound Endpoint Example
+
+The Kafka inbound endpoint of WSO2 ESB acts as a message consumer. It creates a connection to ZooKeeper and requests messages for either a topic/s or topic filters.
+
+## What you'll build
+This sample demonstrates how one way message bridging from Kafka to HTTP can be done using the inbound kafka endpoint.
+
+See [Configuring kafka connector](kafka-connector-configuration.md) for more information.
```

Comment:
Shall we add more explanation of what we build here and a diagram as well?

Reply:
Added image for this section.